Story/FableForest.cs

```diff
@@ -0,0 +1,68 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class FableForest
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(3313))
+        {
+            Core.Logger("You have done this storyline");
```

Comment:
this is fine but generally if u want to add it specifiy what storyline is finished also for 0Allstories it's better not added as already 0Allstories have a logger

Reply:
Okay! I will not add a logger to it